tsconfig.json

```diff
@@ -12,7 +16,7 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "react",
+    "jsx": "preserve",
```

Comment:
do you want to make this change, isn't this what messed with your eslint autofixes?

Reply:
I keep rejecting these changes and it keeps automatically making them... maybe you can help me sometime figure this out.